cmd/cadvisor.go

```diff
@@ -91,6 +93,7 @@ var (
 		container.ReferencedMemoryMetrics:        struct{}{},
 		container.CPUTopologyMetrics:             struct{}{},
 		container.ResctrlMetrics:                 struct{}{},
+		container.VmStatMetrics: 				  struct{}{},
```

Comment:
please extend tests in https://github.com/google/cadvisor/blob/master/cmd/cadvisor_test.go